<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWidgetAddLayer</class>
 <widget class="QStackedWidget" name="StackedWidgetAddLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{

}</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="pageScreen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
border: none;
color: rgb(200, 200, 200);
border: 0px  solid  rgb(64, 64, 64);
padding-top:0px;
}

QToolButton:hover
{
color: rgb(230, 230, 230);
border: 1px  solid  rgb(200, 200, 200);
background-color: rgb(80, 80, 80);
}
QToolButton:disabled
{
color: rgb(100, 100, 100);
border: 0px  solid  rgb(0, 0, 0);
background-color: rgb(0, 0, 0),;
}

QToolButton:checked
{
color: rgb(250, 250, 250);
border: 0px  solid  rgb(250, 250, 250);
background-color: rgb(100, 100, 100);
}
QToolButton:checked:hover
{
color: rgb(230, 230, 230);
border: 1px  solid  rgb(200, 200, 200);
background-color: rgb(80, 80, 80);
}
QToolButton:checked:disabled
{
color: rgb(100, 100, 100);
border: 0px  solid  rgb(0, 0, 0);
background-color: rgb(0, 0, 0),;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutScreen">
    <item>
     <widget class="QToolButton" name="toolButtonScreenArea">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>屏幕区域/窗口</string>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>80</height>
       </size>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QWidget" name="widgetCameraFrame" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCameraViewfinder" name="widgetCamera" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBoxCameras"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonAddCamera">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonCameraSizeFps">
      <property name="text">
       <string>----</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageMedia">
   <property name="styleSheet">
    <string notr="true">QToolButton
{
border: none;
color: rgb(200, 200, 200);
border: 0px  solid  rgb(64, 64, 64);
padding-top:0px;
}
QToolButton:hover
{
color: rgb(230, 230, 230);
border: 1px  solid  rgb(200, 200, 200);
background-color: rgb(80, 80, 80);
}
QToolButton:disabled
{
color: rgb(100, 100, 100);
border: 0px  solid  rgb(0, 0, 0);
background-color: rgb(0, 0, 0),;
}

QToolButton:checked
{
color: rgb(250, 250, 250);
border: 0px  solid  rgb(250, 250, 250);
background-color: rgb(100, 100, 100);
}
QToolButton:checked:hover
{
color: rgb(230, 230, 230);
border: 1px  solid  rgb(200, 200, 200);
background-color: rgb(80, 80, 80);
}
QToolButton:checked:disabled
{
color: rgb(100, 100, 100);
border: 0px  solid  rgb(0, 0, 0);
background-color: rgb(0, 0, 0),;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="toolButtonAddPicture">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>图片文件……</string>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
